### PR TITLE
Enhancement: Added Code from Image as Text

### PR DESCRIPTION
**Pull request by theaadya**
I've written out the code that was previously only available as an image in `CONTRIBUTORS_QUICK_START_VSCODE.rst`, making it more convenient for users as they don't have to manually transcribe it from the image. Please review. 

---

### CODE REVIEW

**Review by hussein-awala — APPROVED** [review]
👍

File: CONTRIBUTORS_QUICK_START_VSCODE.rst [hunk 1]
```diff
@@ -92,6 +92,12 @@ Setting up debugging
 
   - Change ``"program"`` to point to an example dag and add ``"env"`` and ``"python"`` fields to the new Python configuration
 
+    ``"program": "${workspaceFolder}/files/dags/example_bash_operator.py"``
+
+    ``"env": {"PYTHONUNBUFFERED": "1", "AIRFLOW__CORE__EXECUTOR": "DebugExecutor"}``
+
+    ``"python": "${env:HOME}/.pyenv/versions/airflow/bin/python"``
+
```
Comment (Lee-W) [inline CONTRIBUTORS_QUICK_START_VSCODE.rst]:
```suggestion
.. code-block:: json

   {
       "configurations": [
           "program": "${workspaceFolder}/files/dags/example_bash_operator.py",
           "env": {
               "PYTHONUNBUFFERED": "1",
               "AIRFLOW__CORE__EXECUTOR": "DebugExecutor"
            },
            "python": "${env:HOME}/.pyenv/versions/airflow/bin/python"
       ]
   }

```

not sure whether it's better this way, but I might try something like this

**Review by Lee-W — APPROVED** [review]

**Review by utkarsharma2 — APPROVED** [review]

File: CONTRIBUTORS_QUICK_START_VSCODE.rst (hunk 1 above)
Comment (theaadya) [reply CONTRIBUTORS_QUICK_START_VSCODE.rst]: You're right, it does seem better this way. I'll update. 👍

**Review by amoghrajesh — APPROVED** [review]
Good work @theaadya
LGTM +1

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Cool!

**Pull request merged**